src/components/MostPopular/MostPopular.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import HttpService from '../../service/HttpService/httpService';
+import ProductItem from './ProductItem';
+
+class MostPopular extends Component {
+  state = {
+    products: [],
+  }
+
+  async downloadProducts() {
+    const userApi = new HttpService();
+    const response = await userApi.get('http://localhost:4000/most-popular');
+    this.setState({ products: response.data });
```

Comment:
what happens when you get an error in the response?

Reply:
the component won't rendered

Reply:
please add some verify for response -> if (response && response.data) {... }